addons/binding/org.openhab.binding.dscalarm/ESH-INF/thing/keypad.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dscalarm"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+     <thing-type id="keypad">
+
+	    <supported-bridge-type-refs>
+            <bridge-type-ref id="envisalinkbridge" />
+            <bridge-type-ref id="it100bridge" />
+        </supported-bridge-type-refs>
+
+        <label>DSC Alarm Keypad</label>
+        <description>Represents the central administrative unit of the DSC Alarm System.</description>
+
+        <channels>
+            <channel id="keypad_ready_led" typeId="keypad_ready_led"/>
```

Comment:
All your channels seem to be of the same type - an signalling led with is on/off/flashing.
Since a few weeks, you can add label and descriptions directly to the channel definitions here, so there is probably no need to define different channel types, but one should be enough.

Try something like
```
<channel id="keypad_ready_led" typeId="led">
  <label>Keypad Armed LED</label>
  <description>Keypad Armed LED (0=Off, 1=On, 2=Flashing)</description>
</channel>
```